config/coreboot-t400-maximized-debug.config

```diff
@@ -0,0 +1,15 @@
+CONFIG_VENDOR_LENOVO=y
+CONFIG_CBFS_SIZE=0x7FCFFF
+CONFIG_IFD_BIN_PATH="@BLOB_DIR@/xxx0/ifd.bin"
+CONFIG_GBE_BIN_PATH="@BLOB_DIR@/xxx0/gbe.bin"
+CONFIG_HAVE_IFD_BIN=y
+CONFIG_BOARD_LENOVO_T400=y
+CONFIG_LINUX_COMMAND_LINE="intel_iommu=igfx_off quiet"
```

Comment:
Was different in past commit. I do not remember why different OS kernel tweaks were passed before.